web/src/main/java/edu/gemini/lch/web/app/windows/night/BlanketClosuresTable.java

```diff
@@ -40,7 +42,7 @@
 
     public BlanketClosuresTable(final NightWindow parent) {
         this.parent = parent;
-        this.timeFormatter = new TimeFormatter(DateTimeZone.UTC);
+        this.timeFormatter = new TimeFormatter(ZoneId.of("UTC"));
```

Comment:
Why `UTC` here instead of system default?

Reply:
For this, I'm not sure. At this point, the default could be set to Chile time, whereas Florian specifically was working with UTC, so I tried to maintain the same time zones that he was using in his code at the same places to avoid any strange bugs. As I went down the rabbit hole, everything I've read suggests that the transition from joda to Java 8 is nowhere near as simple as it is portrayed by some people, and it is recommended against unless the benefits are substantial.

I think the benefits aren't particularly substantial, unfortunately: nowhere near as much as they would have been had we moved past our JRE blockade that forces that irritating time zone update process. We probably should have spent a bit more time there.